be/src/common/configbase.cpp

```diff
@@ -432,6 +433,20 @@ Status set_config(const std::string& field, const std::string& value, bool need_
                                 it->second.type);
 }
 
+Status set_fuzzy_config(const std::string& field, const std::string& value) {
+    return set_config(field, value, false, true);
+}
+
+void set_fuzzy_configs() {
+    // random value true or false
+    Status s = set_fuzzy_config("enable_storage_vectorization",
```

Comment:
Not fuzzy this variable, maybe some code could not run under non-vectorization engine.



I think you could fuzzy disable storage page cache.